src/Routes.js

```diff
@@ -80,6 +88,28 @@ export const AppRoutes = () => (
           />
         }
       />
+      <Route
```

Comment:
Is there a plan to use a feature flag for this?

Reply:
Yes, forgot to add. Will do it. Thanks